src/test/java/fr/openent/viescolaire/service/impl/DefaultGroupingServiceTest.java

```diff
@@ -126,4 +126,23 @@ public void TestDeleteGrouping(TestContext ctx) {
         defaultGroupingService.deleteGrouping(groupingTestId);
     }
 
+    @Test
+    public void TestDeleteGroupingAudience(TestContext ctx) {
+        Async async = ctx.async();
+        String queryExpected = "DELETE FROM " + tableRel + " WHERE " + tableRel + ".grouping_id = ? AND " + tableRel + ".student_division_id = ?";
```

Comment:
Simplifiable par : "DELETE FROM " + tableRel + " WHERE grouping_id = ? AND student_division_id = ?";

Reply:
Magic string pour les tableRel ?